packages/transport-circuit-relay-v2/src/transport/reservation-store.ts

```diff
@@ -75,6 +76,13 @@ export interface ReservationStoreEvents {
   'relay:removed': CustomEvent<PeerId>
 }
 
+const configValidator = object({
+  discoverRelays: number().integer().min(0).default(0),
+  maxReservationQueueLength: number().integer().min(0).default(100),
+  reservationCompletionTimeout: number().integer().min(0).default(10000),
+  reservationConcurrency: number().integer().min(0).default(DEFAULT_RESERVATION_CONCURRENCY)
```

Comment:
We have some defaults defined as constants, some as literals - can it be one or the other please, probably named constants.